README.md

```diff
@@ -21,27 +21,23 @@ Table of contents
 
 # Join the #ramp-up channel on Puppet Community Slack
 
-Our [Puppet Community Slack](http://slack.puppet.com) is a great way to interact with other Puppet users.  The #ramp-up channel is specifically for users talking about starting with PE and using this repository.
-
-Other channels in the Puppet Community Slack are great for asking general Puppet questions as well.
+Our [Puppet Community Slack](http://slack.puppet.com) is a great way to interact with other Puppet users.  The #ramp-up channel is specifically for users who are using this repository who are new to Puppet Enterprise. Other channels in the Puppet Community Slack are great for asking general Puppet questions.
```

Comment:
looks like we're missing a word "who are using this repository who are new to Puppet Enterprise"